site/content/docs/03-run-time.md

```diff
@@ -254,6 +254,27 @@ Events dispatched from child components can be listened to in their parent. Any
 <Child on:notify="{callbackFunction}"/>
 ```
 
+---
+
+Events can also be cancelable by passing a third parameter to the dispatch function. Event listeners can then call `event.preventDefault()` to control the dispatcher's return value.
```

Comment:
```suggestion
Events can be cancelable by passing a third parameter to the dispatch function. Event listeners can then call `event.preventDefault()` to control the dispatcher's return value.
```
Also not sure about the last part of the second sentence. I really like the description in the changelog, maybe that can be reused here somehow

Reply:
Made some adjustments in the latest commit. I realized the compact description of "return a boolean depending on whether the event is cancelled" sort-of implies that "if cancelled, it returns true", so I elaborated more in the docs.